ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -46,6 +46,8 @@ open class SideTabBar: UIView {
                 avatarView.avatarSize = .medium
                 avatarView.translatesAutoresizingMaskIntoConstraints = false
                 avatarView.overrideAccessibilityLabel = "Accessibility.LargeTitle.ProfileView".localized
+                avatarView.showsLargeContentViewer = true
+                avatarView.largeContentTitle = avatarView.overrideAccessibilityLabel
```

Comment:
Do we want this to just be the default? that avatarView.overrideAccessibilityLabel maps to avatarView.largeContentTitle ?

Reply:
i guess i don't understand you comment. would you please clarify?

Reply:
Do we want to make it so that setting `AvatarView. avatarView.overrideAccessibilityLabel` internally sets `avatarView.largeContentTitle` in the setter?

Reply:
no. not all avatarview is meant to have a large content viewer. if you have a personacell, the large content isn't useful. these are few cases that avatarview is a navigation chrome control

Reply:
I see, thanks!